index.js

```diff
@@ -0,0 +1,84 @@
+// dependencies
+var async = require('async');
+var AWS = require('aws-sdk');
+var util = require('util');
+
+var mailparse = require('./parse.js');
+// get reference to S3 client 
+var s3 = new AWS.S3();
+
+
+
+exports.handler = function(event, context, callback) {
+    // Read options from the event.
+    var eventBody = event.Records[0].body;
+    sources3 = JSON.parse(eventBody).Records[0].s3;
```

Comment:
@sudhirj @sivachandran Should we be adding the S3 event here? I think routing it via SQS is a better option.

Reply:
It seems we are processing the SQS event. We need to delete the message from the queue after processing @sanjeevsiva17.

Reply:
According to SQS documentation, the message will delete automatically. 
https://docs.aws.amazon.com/lambda/latest/dg/with-sqs.html

@sudhirj @sivachandran @maknahar